extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -1,5 +1,8 @@
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index';
-@import 'skins/shared/color.scss';
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+@import 'skins/oasis/css/core/breakpoints-variables';
 
 .WikiaPage .page-header {
```

Comment:
@latata I needed more specific selector to overwrite some other oasis styling without using !important.